components/composed/ConfirmModalBody/ConfirmModalBody.tsx

```diff
@@ -0,0 +1,37 @@
+import * as Styled from "./ConfirmModalBody.styles";
+import { ButtonControlConfirm } from "components/atomic";
+
+type ButtonControlConfirmProps = React.ComponentProps<
+  typeof ButtonControlConfirm
+>;
+
+const ConfirmModalBody = ({
+  modalBody: body,
+  handleClose,
+  onClick: onConfirm,
+  actionLabel,
+}: ConfirmModalBodyProps) => {
+  return (
+    <>
+      <p className="t-copy-sm">{body}</p>
+      <Styled.ButtonWrapper>
+        <Styled.ConfirmButton onClick={onConfirm}>
+          {actionLabel ?? "Confirm"}
```

Comment:
These buttons need i18n keys (Apologies if this was already on your list of todos 😅 )